CODEOWNERS

```diff
@@ -66,6 +69,14 @@ android/ @ahumesky
 /tools/test/collect_cc_coverage.sh @lberki
 /tools/test/CoverageOutputGenerator/ @lberki
 
+# Configurability
+
+/src/main/java/com/google/devtools/build/lib/analysis @bazelbuild/configurability
+/src/main/java/com/google/devtools/build/lib/rules/platform @bazelbuild/configurability
+/src/test/java/com/google/devtools/build/lib/analysis @bazelbuild/configurability
+/src/test/java/com/google/devtools/build/lib/rules/platform @bazelbuild/configurability
+/tools/platforms @bazelbuild/configurability @bazelbuild/configurability
```

Comment:
Repeat?

Reply:
It only look that way  the paths switch  /main/ => /test/.
It's the broken java convention of not having the tests in the same directory as the code.

Reply:
I think he means the double team mention on this line.

Reply:
Ah, yes. It's hard to  see your own typos.

Reply:
Fixed in the merge